.github/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "monthly"
+      day: "sunday"
+      time: "19:00"
+      timezone: "US/Pacific"
+    # Labels on pull requests for security and version updates
+    labels:
+      - "npm dependencies"
+    reviewers:
+      - "zoobot"
+    rebase-strategy: "auto"
+    # Allow up to 10 open pull requests for pip dependencies
```

Comment:
I see you changed the limit to 5 below, so this comment should say `Allow up to 5 open pull requests`.